source/dbo/Stored Procedures/Unmatched Specimens/uspGenerateUnmatchedSpecimensLinkedNotifications.sql

```diff
@@ -23,42 +23,45 @@ BEGIN TRY
 	--Exclude any matches that have previously been rejected (otherwise every rejection has an accompanying match)
 	LEFT JOIN #PreviousMatch pm on pm.NotificationID = mr.rclientsourceID AND pm.ReferenceLaboratoryNumber = mr.clientsourceID
 	LEFT JOIN #PreviousPossibleMatch ppm on ppm.NotificationID = mr.rclientsourceID AND ppm.ReferenceLaboratoryNumber = mr.clientsourceID
+	JOIN [$(NTBS_Specimen_Matching)].dbo.MatchingConfiguration mc ON mr.[weight] < mc.AutoMatch
 	WHERE pm.EventType IS NULL AND ppm.EventType IS NULL
-	AND mr.[weight] < 34.50
+	-- DACPAC!
 
-	CREATE TABLE #AllMatches (ReferenceLaboratoryNumber NVARCHAR(50), NotificationID int, EventType NVARCHAR(50))
-	INSERT INTO #AllMatches SELECT * FROM #PreviousMatch
-	INSERT INTO #AllMatches SELECT * FROM #PreviousPossibleMatch
-	INSERT INTO #AllMatches SELECT * FROM #PoorQualityMatch
+	CREATE TABLE #AllLinks (ReferenceLaboratoryNumber NVARCHAR(50), NotificationID int, EventType NVARCHAR(50))
+	INSERT INTO #AllLinks SELECT * FROM #PreviousMatch
+	INSERT INTO #AllLinks SELECT * FROM #PreviousPossibleMatch
+	INSERT INTO #AllLinks SELECT * FROM #PoorQualityMatch
 
 	INSERT INTO [dbo].[UnmatchedSpecimensLinkedNotifications]
-	SELECT am.ReferenceLaboratoryNumber
+	SELECT al.ReferenceLaboratoryNumber
 		,CASE
-			WHEN am.EventType = 'RejectPotentialMatch' THEN 'Possible match rejected by ' + u.UserDisplayName + ' on ' + FORMAT(u.RejectionTime, 'dd MMM yyyy')
-			WHEN am.EventType = 'Unmatch' THEN 'Confirmed match rejected by ' + u.UserDisplayName + ' on ' + FORMAT(u.RejectionTime, 'dd MMM yyyy')
-			WHEN am.EventType = 'PoorQualityMatch' THEN 'Poor quality potential match'
+			WHEN al.EventType = 'RejectPotentialMatch' THEN 'Possible match rejected by ' + rs.UserDisplayName + ' on '
+			WHEN al.EventType = 'Unmatch' THEN 'Confirmed match rejected by ' + rs.UserDisplayName + ' on '
+			WHEN al.EventType = 'PoorQualityMatch' THEN 'Poor quality potential match'
```

Comment:
I think it would be better to keep the type and the date in the table, and then do all of the formatting in Power BI, rather than half here and half there.